# This message will contain all information about road sections
# It needs to be published via a publisher or a service.
geometry_msgs/Point32[] left_line
geometry_msgs/Point32[] middle_line
geometry_msgs/Point32[] right_line


############################
# TODO: Add more variables #
############################ 
